compiler/compiler.go

```diff
@@ -555,7 +595,7 @@ func (c *compileCtx) compileSchema(name string, s *openapi.Schema) (protobuf.Typ
 	switch {
 	case s.Type.Empty() || s.Type.Contains("object"):
 		if ap := s.AdditionalProperties; ap != nil && !ap.IsNil() {
-			return c.compileMap(name, ap)
+			return c.compileMap(name, strings.TrimSuffix(rawName, "Message"), ap)
```

Comment:
Is this in response to some internal styling you are parsing?

Reply:
@jprobinson The variable name inside the wrapper was ending up with `Message` at the end, which does not align with the variable name in the input swagger spec.

Reply:
Sounds fine. As long as the unit tests pass, we good.